qt/aqt/reviewer.py

```diff
@@ -456,6 +457,7 @@ def _shortcutKeys(
             ("Shift+v", self.onRecordVoice),
             ("o", self.onOptions),
             ("i", self.on_card_info),
+            ("Ctrl+i", self.on_previous_card_info),
```

Comment:
Alt feels like a more natural choice for an alternate/modified action that ctrl/cmd to me - was there a technical reason/conflict that prevented you from using it?

Reply:
Alt is just hard for my hands to press, then, I press first using Ctrl and if not available go to Alt. But on this case, other than my hand issues, I think Alt could be better to easily confuse a user that things Ctrl+i is for the current card.